crafting_bench.lua

```diff
@@ -1,3 +1,7 @@
+local get_meta = minetest.get_meta
+local get_node_timer = minetest.get_node_timer
+local pos_to_string = minetest.pos_to_string
```

Comment:
FWIW, there is little to negligible performance difference to justify this boilerplate.